File: model/permiso/fitosanitario/PermisoFitosanitarioCrearRq.java
/**
 * 
 */
package org.sieca.everis.pdcc.model.permiso.fitosanitario;

import org.sieca.everis.pdcc.model.permiso.Encabezado;
import org.sieca.everis.pdcc.model.util.RequestBody;

/**
 * @author ytorregl
 *
 */
public class PermisoFitosanitarioCrearRq extends RequestBody{
	
    private Encabezado encabezado;
    private Datos datos;
    private String tipoOperacion;
	/**
	 * @return the encabezado
	 */
	public Encabezado getEncabezado() {
		return encabezado;
	}
	/**
	 * @param encabezado the encabezado to set
	 */
	public void setEncabezado(Encabezado encabezado) {
		this.encabezado = encabezado;
	}
	/**
	 * @return the datos
	 */
	public Datos getDatos() {
		return datos;
	}
	/**
	 * @param datos the datos to set
	 */
	public void setDatos(Datos datos) {
		this.datos = datos;
	}
	/**
	 * @return the tipoOperacion
	 */
	public String getTipoOperacion() {
		return tipoOperacion;
	}
	/**
	 * @param tipoOperacion the tipoOperacion to set
	 */
	public void setTipoOperacion(String tipoOperacion) {
		this.tipoOperacion = tipoOperacion;
	}
    
    

}

File: model/permiso/fitosanitario/PermisoFitosanitarioCrearRq.php
/*/
@Yimis Torreglosa Diaz
/*/
<?php class PermisoFitosanitarioCrearRq
{
    //VARIABLES
    private $Encabezado;
    private $Datos;
    private $tipoOperacion;

    //CONSTRUCTOR
    public function __construct()
    {
        //
    }

    // METODOS GETTERS y SETTERS

    /**
     * Get the value of Encabezado
     */

    public function getEncabezado()
    {
        return $this->Encabezado;
    }

    /**
     * Set the value of Encabezado
     *
     * @return  self
     */

    public function setEncabezado($Encabezado)
    {
        $this->Encabezado = $Encabezado;

        return $this;
    }

    /**
     * Get the value of Datos
     */

    public function getDatos()
    {
        return $this->Datos;
    }

    /**
     * Set the value of Datos
     *
     * @return  self
     */

    public function setDatos($Datos)
    {
        $this->Datos = $Datos;

        return $this;
    }

    /**
     * Get the value of tipoOperacion
     */

    public function getTipoOperacion()
    {
        return $this->tipoOperacion;
    }

    /**
     * Set the value of tipoOperacion
     *
     * @return  self
     */

    public function setTipoOperacion($tipoOperacion)
    {
        $this->tipoOperacion = $tipoOperacion;

        return $this;
    }
} ?>

